java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/ArrowFlightJdbcDriver.java

```diff
@@ -58,7 +59,7 @@ public class ArrowFlightJdbcDriver extends UnregisteredDriver {
     // Netty requires some extra properties to unlock some native memory management api
     // Setting this property if not already set externally
     // This has to be done before any netty class is being loaded
-    final String key = "cfjd.io.netty.tryReflectionSetAccessible";
+    final String key = "io.netty.tryReflectionSetAccessible";
```

Comment:
Do we want to support both the old and the new keys to minimize potential compatibility breaks?

Reply:
@lidavidm do you mean testing/shading both `cfjd` and regular shading ? I think it could be confusing for users. Just wondering.

Reply:
The main use of this property is to be set directly by the driver (not the users) in order for the driver to actually work. Checking if it is set sounds more like a debugging capability.